scripts/phenology/phenology_difference.yml

```diff
@@ -0,0 +1,28 @@
+script: phenology_difference.R
+name: Phenology difference
+timeout: 1440
+author: 
+  - name: Jory Griffith
+license: CC-BY
+description: Script to calculate the difference in the layers of the phenology raster layers from the start to end year
```

Comment:
What does it give you to calculate this? Maybe a small rationale behind why we are doing this and how to interpret results.